content/en/serverless/distributed_tracing/serverless_trace_merging.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Serverless Trace Merging
+kind: documentation
+description: 'Augment AWS X-Ray Tracing with Datadog APM'
+further_reading:
+    - link: 'serverless'
+      text: 'Serverless Monitoring with Datadog'
+---
+
+{{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;" >}}
+
+## Use Cases
+
+Serverless trace merging is required to see a single, connected trace when you configure both Datadog's tracing libraries (`dd-trace`) and AWS X-Ray tracing libraries in your application. If you aren't sure which tracing library to use, [read this page on choosing your tracing library][1].
+
+There are two primary reasons for instrumenting both `dd-trace` and AWS X-Ray tracing libraries:
+- In a serverless-first environment, your application requires AWS X-Ray active tracing in AWS managed services such as API Gateway or Step Functions, you are tracing your Lambda functions with `dd-trace`, and you want to visualize connected traces across your serverless resources.
```

Comment:
I personally prefer `In a Lambda environment` over `In a serverless-first environment` for clarity.